docs/whats-new.asciidoc

```diff
@@ -13,140 +13,194 @@ Other versions: {security-guide-all}/8.5/whats-new.html[8.5] | {security-guide-a
+
+{ml-cap} job names are used throughout the {security-app} instead of job IDs so you can quickly know which {ml-cap} jobs you're interacting with.
+
+*Prebuilt rule updates*
+
+Check out the {security-guide}/prebuilt-rule-8-7-1-prebuilt-rules-8-7-1-summary.html[latest updates] to prebuilt rules. To download the latest updates, refer to the instructions in {security-guide}/rules-ui-management.html#download-prebuilt-rules[Download latest Elastic prebuilt rules]. 
 
 [float]
-== New Data Exfiltration Detection (DED) integration
+== New integrations 
 
-The https://docs.elastic.co/integrations/ded[DED integration package] protects your data from being exfiltrated to external web services (technical preview only and requires a Platinum subscription).
+The following security integrations have been added in 8.7: 
+
+* AuditD
+* Content Delivery Network
+* Credential Management
+* Database
+* EDR/XDR
+* Email Security
+* Firewall
+* Identity & Access Management
+* Threat Intelligence
+* Vulnerability Management
+
+The Integrations page also has improved categorization and search functionality.
 
 [role="screenshot"]
-image::whats-new/images/8.6/DED-integration.png[DED integration]
+image::whats-new/images/8.7/integrations.png[Integrations page]
+
 
 [float]
-== Entity Analytics dashboard enhancements
+== Role-based access control (RBAC) for response actions
 
-The following enhancements have been made to the {security-guide}/detection-entity-dashboard.html[Entity Analytics dashboard] to improve efficiency in responding to emerging threats: 
+New Endpoint management feature privileges in {kib} allow admins to have more granular control over {security-guide}/endpoint-management-req.html[{elastic-defend} features]. Role-based access can now be managed for the following:
 
-* Hover over a host name or user name to display inline actions: *Add to timeline investigation*, which adds the selected value to a new Timeline, and *Copy to Clipboard*, which copies the host name value for you to paste later.
-* Launch Timeline by clicking the number link in the *Alerts* column. 
-* Install and run a {ml} job directly from the *Notable Anomalies* table  by clicking *Run job*. 
+* Hosts running {elastic-defend}
+* Trusted applications
+* Host isolation exceptions
+* Blocklist entries
+* Event filters
+* {elastic-defend} policies
+* File-related operations in the response console
 
 [role="screenshot"]
-image::whats-new/images/8.6/entity-dashboard.png[Entity Analytics Dashboard]
+image::whats-new/images/8.7/endpoint-privileges.png[Role-based access controls, 75%]
 
 [float]
-== Exceptions enhancements
+== Response console action enhancements
 
-The following enhancements improve the exceptions feature's UI and simplifies management: 
+A new `get file` {security-guide}/response-actions.html[response console] action allows you to retrieve and download a file from a host. Files are downloaded in a password-protected `.zip` archive to prevent the file from running.
```

Comment:
```suggestion
A new `get file` {security-guide}/response-actions.html[response console] action allows you to retrieve and download a file from a host. Files are downloaded in a password-protected `.zip` archive to prevent them from running.
```